newbank/server/NewBankClientHandler.java

```diff
@@ -59,37 +58,31 @@ public void run() {
 		try {
 
 			// ask for user name and password
-			out.println("Enter Username");
-			String userName = in.readLine();
-			out.println("Enter Password");
-			String password = in.readLine();
-			out.println("Checking Details...");
-
-			// authenticate user and get customer ID token from bank for use in subsequent requests
-			CustomerID customer = bank.checkLogInDetails(userName, password);
-
-			// if the user is authenticated then get requests from the user and process them
-			if(customer != null) {
-
+			//Salutation
+			out.println("Welcome to NewBank");
+			String entry = "";
```

Comment:
Do we use ‘entry’?

Reply:
No, we do not use entry, I can remove this.